src/vs/platform/extensionManagement/common/extensionManagement.ts

```diff
@@ -374,6 +384,8 @@ export interface IExtensionTipsService {
 	getKeymapRecommendations(): string[];
 	getKeywordsForExtension(extension: string): string[];
 	getRecommendationsForExtension(extension: string): string[];
+	refreshAllIgnoredRecommendations(): TPromise<void>;
+	getAllIgnoredRecommendations(): IIgnoredRecommendations;
```

Comment:
If we update `ExtensionsEditor` to use the `onRecommendationChange` event and move the `ignoreRecommendation` function from `ExtensionWorkbenchService` to `ExtensionsTipsService`, would we still need to have these 2 functions as public and therefore defined in the interface?